.github/workflows/python.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Distribute PhotonLibPy
+
+permissions:
+  id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8  # CHose earliest that robotpy supports
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+          pip install robotpy
+
+      - name: Build wheel
+        working-directory: ./photon-lib/py
+        run: |
+          git fetch --tags --force
+          python setup.py sdist bdist_wheel
```

Comment:
I'd recommend using `pipx run build`.

Reply:
![image](https://github.com/PhotonVision/photonvision/assets/4583662/622e9084-2903-47d6-a7a3-add0da8ff1bd)
pipx is currently being very sad about installing for me.... thoughts on leaving this one for later?

Reply:
Hmm, [pipx's docs](https://github.com/pypa/pipx) recommend scoop over pip for installing pipx.

Reply:
I'd probably specifically recommend https://github.com/hynek/build-and-inspect-python-package in GitHub Actions though